### DISCUSSION TITLE
How to Fix the Repos I've Made

### DISCUSSION THREAD

**Discussion by raytjos** [community/community#21732] [New to GitHub]
<p>I would like to include the local repository to an existing repo in a GetHub paid private repro which I have access. There is only a readme file currently in the GHrepo. I want to include my local repo in a subdirectory of the GHrepo.</p>
<p>I installed the GHDesktop to setup my local. I have a different GH login for it. I do not see how to get the GHDesktop to recognized the private GH repro.</p>
<p>What do I need to do to include my local repo up on paid private GH repo?</p>
<p>I have three other coworkers that will need to use the GHrepo. I am not sure of the terminalogy, but would they clone that GHrepo to their local machines?  How would they do that?</p>
<p>How would they setup GHDesktop?</p>
<p>I mistakenly created a parallel GHrepo on this private account. It has one file I used to test with. How to I get rid/delete this extra GHrepo?</p>
<p>Thank you</p>
<p>Ray</p>

**Comment by steveward** [comment, marked as answer]
<p>Hi <a class="mention" href="/raytjos">@raytjos</a>!</p>
<p>&gt;I would like to include the local repository to an existing repo in a GetHub paid private repro which I have access. There is only a readme file currently in the GHrepo. I want to include my local repo in a subdirectory of the GHrepo.</p>
<p>Is the local project already an existing repository?</p>
<p>It sounds like you want your local repository and the remote repository that is currently on GitHub to become merged together. Here’s what I would recommend doing:</p>
<ol>
<li>
<p>Clone the remote repository using GitHub Desktop by going to <code>File</code> &gt; <code>Clone Repository</code> &gt; <code>GitHub.com</code> and selecting the repository from the list.</p>
</li>
<li>
<p>Once the repository has been cloned to GitHub Desktop go to the menu and select <code>Repository</code> &gt; <code>Open in Explorer/Finder</code></p>
</li>
<li>
<p>Add your project files to this location </p>
</li>
<li>
<p>Switch back to GitHub Desktop</p>
</li>
<li>
<p>In the <code>Changes</code> tab you will see that your newly added project files have been added. You can now commit those changes in GitHub Desktop.</p>
</li>
<li>
<p>Once those changes have been committed click the <code>Push origin</code> button to push the changes up to your remote repository on GitHub</p>
</li>
</ol>
<p>Your local and remote repository should now be synced.</p>
<p>&gt;I installed the GHDesktop to setup my local. I have a different GH login for it. I do not see how to get the GHDesktop to recognized the private GH repro.</p>
<p>I’m not sure I understand the issue – do you have multiple GitHub accounts? You can change accounts by going to <code>File</code> &gt; <code>Options</code> &gt; <code>Accounts</code>.</p>
<p>&gt;I have three other coworkers that will need to use the GHrepo. I am not sure of the terminalogy, but would they clone that GHrepo to their local machines?  How would they do that?</p>
<p>Your coworkers could clone the repository to their local machines using GitHub Desktop or the command line. You will need to add them as collaborators – you can read how to do that here:</p>
<p><a href="https://help.github.com/en/articles/inviting-collaborators-to-a-personal-repository" rel="nofollow noopener">https://help.github.com/en/articles/inviting-collaborators-to-a-personal-repository</a></p>
<p>&gt;How would they setup GHDesktop?</p>
<p>They can install GitHub Desktop from <a href="http://desktop.github.com" rel="nofollow noopener">http://desktop.github.com</a> and clone the repository there.</p>
<p>&gt;I mistakenly created a parallel GHrepo on this private account. It has one file I used to test with. How to I get rid/delete this extra GHrepo?</p>
<p>You can read how to delete a repository on GitHub here:</p>
<p><a href="https://help.github.com/en/articles/deleting-a-repository" rel="nofollow noopener">https://help.github.com/en/articles/deleting-a-repository</a></p>
<p>I hope that is helpful. Let me know if you have any other questions.</p>

**Comment by lee-dohm** [comment]
<p>Let me see if I can address your various questions:</p>
<ul>
<li>Desktop doesn’t currently support multiple accounts, <a href="https://github.com/desktop/desktop/issues/3707" rel="nofollow noopener">see this issue for details</a>
</li>
<li><a href="https://help.github.com/en/articles/cloning-a-repository" rel="nofollow noopener">How to clone a GitHub repository</a></li>
<li><a href="https://desktop.github.com" rel="nofollow noopener">Download and install GitHub Desktop</a></li>
<li><a href="https://help.github.com/en/articles/deleting-a-repository" rel="nofollow noopener">Deleting a repository</a></li>
</ul>
<p>As for “include your local repo in the paid private GitHub repo”, that depends on what you mean by “include”? If you want to put the contents that you have in your local repository into the paid private GitHub repository, then you can do the following:</p>
<ol>
<li>Clone the paid private GitHub repository to your local machine (see the “How to clone a GitHub repository” instructions above)</li>
<li>Copy the files from your local repo to the local copy of the paid private repository</li>
<li><a href="https://help.github.com/en/desktop/contributing-to-projects/committing-and-reviewing-changes-to-your-project" rel="nofollow noopener">Commit and push the changes</a></li>
</ol>
<p>I hope that helps!</p>

**Comment by raytjos** [comment]
<p>This is now working.  I really appreciate all the help.</p>
<p>There is a potential issue I would like to resolve before I roll this out.  I created a new repository parallel to the existing.  This is in a corporate developer site on GitHub.  I noticed that the repo I created is tagged as private.  The existing repo is not.  May intent was not to limit access to corporate personnel but I am guessing that is what happened. </p>
<p>So my question is ‘should I make this new repo public?’  Since it is inside a corporate domain on GitHub, will it be available to the general public?</p>
<p>What is the best way to assure that all corporate collaborates have access?</p>

**Comment by raytjos** [comment]
<p>Follow-up</p>
<p>It looks like everyone in the corporation that has any rights, has all those rights for this repo per settings/collaboration.  </p>
<p>So the secondary question is whether I should transfer ownership.  There seems to be two options.  I can transfer it to the company-admin or to the company (which may default to the admin).  How may if determine an appropriate route?</p>

**Comment by lee-dohm** [comment]
<p>&gt; So the secondary question is whether I should transfer ownership.</p>
<p>From your description it sounds like there are two repositories:</p>
<ul>
<li>raytjos/some-repo</li>
<li>some-company/some-repo</li>
</ul>
<p>and that you copied the contents from <code>raytjos/some-repo</code> to <code>some-company/some-repo</code>. And I assume that <code>some-company/some-repo</code> is the one that you were previously saying was private?</p>
<p>If all of the above is correct, then I have two questions:</p>
<ul>
<li>Which repository is it you’re asking about transferring the ownership of?</li>
<li>If some-company already has the code in some-company/some-repo, then why would you need to transfer ownership of raytjos/some-repo?</li>
</ul>
<p>Let us know!</p>
